### PR TITLE
Lua_api.txt: Split long lines part 5, fix hardware colouring docs

### PR DESCRIPTION
**Pull request by paramat**
:P~~

### CODE REVIEW

**Comment by paramat** [comment]
0d8cb8433a2e310a193a51b0cd49b117a6d899ce

File: doc/lua_api.txt [hunk 1]
```diff
@@ -580,25 +580,25 @@ index of the pixel to use.
...
   Examples:
     * `param2 = 17` is 2 * 8 + 1, so the rotation is 1 and the third (= 2 + 1)
       pixel will be picked from the palette.
     * `param2 = 35` is 4 * 8 + 3, so the rotation is 3 and the fifth (= 4 + 1)
       pixel will be picked from the palette.
-* `drawtype = "colorfacedir"` for nodes which use the first
+* `paramtype2 = "colorfacedir"` for nodes which use the first
   three bits of `param2` for palette indexing. The remaining
-  five bits are describing rotation, as in `facedir` draw type.
+  five bits are describing rotation, as in `facedir` drawtype.
```
Comment (Thomas--S) [inline doc/lua_api.txt]: Shouldn't this be `paramtype2`, too?

File: doc/lua_api.txt [hunk 2]
```diff
@@ -580,25 +580,25 @@ index of the pixel to use.
...
-* `drawtype = "color"` for nodes which use their full `param2` for
+* `paramtype2 = "color"` for nodes which use their full `param2` for
   palette indexing. These nodes can have 256 different colors.
   The palette should contain 256 pixels.
-* `drawtype = "colorwallmounted"` for nodes which use the first
+* `paramtype2 = "colorwallmounted"` for nodes which use the first
   five bits (most significant) of `param2` for palette indexing.
   The remaining three bits are describing rotation, as in `wallmounted`
-  draw type. Division by 8 yields the palette index (without stretching the
+  drawtype. Division by 8 yields the palette index (without stretching the
```
Comment (Thomas--S) [inline doc/lua_api.txt]: Same here

Comment (SmallJoker) [reply doc/lua_api.txt]: You're right. These two are indeed values for `paramtype2`.

Comment (paramat) [reply doc/lua_api.txt]: Thanks will fix.

File: doc/lua_api.txt (hunk 1 above)
Comment (paramat) [reply doc/lua_api.txt]: Yes.

**Pull request closed without merging**